src/lib.js

```diff
@@ -257,7 +261,7 @@ export default class Migrator {
           name: migrationName,
           createdAt: timestamp
         });
-        return createdMigration.toJSON();
+        return createdMigration;
```

Comment:
I'd much prefer if we leave the toJSON() in and put an before the MigrationiModel.create. This way we can still reliably exclude the id fields and include the `filename` virtual field